libs/sdk-ui-dashboard/src/presentation/saveAs/DefaultSaveAsDialog/useSaveAs.ts

```diff
@@ -57,6 +57,7 @@ export interface UseSaveAsProps {
 }
 
 export const useSaveAs = (props: UseSaveAsProps): UseSaveAsResult => {
+    const [isDashboardSaving, setIsDashboardSaving] = useState(false);
```

Comment:
actually.. one thing.. please add TODO that this should be coming from the model..

Reply:
Will do